esp/scm/WsDeploy.ecm

```diff
@@ -568,7 +568,7 @@ ESPresponse [exceptions_inline, encode(0)] GetSummaryResponse
     string XmlStr;
 };
 
-ESPservice [exceptions_inline("xslt/exceptions.xslt")] WsDeploy
+ESPservice [requiredaccess("NONE"), exceptions_inline("xslt/exceptions.xslt")] WsDeploy
```

Comment:
The "NONE" keyword makes more sense if the attribute becomes more explicit (and accurate)... above I threw out a very verbose option of "autoFeatureAuth" which I like because it is self documenting, I'll likely implement this change unless you have a strong opinion against it... @afishbeck